test/graph/test_graph.py

```diff
@@ -132,6 +135,18 @@ def test_next_neighbors():
     assert graph1.n_edges == 3 * graph2.n_edges
 
 
+def test_custom_edges():
+    graph = nk.graph.KitaevHoneycomb(extent=[3, 3])
+    for i in range(3):
+        assert len(graph.edges(i)) == 9
```

Comment:
This gives a deprecation warning otherwise.
```suggestion
        assert len(graph.edges(filter_color=i)) == 9
```